test/Continuity.lean

```diff
@@ -0,0 +1,24 @@
+import Mathlib.Topology.Basic
+
+variable [TopologicalSpace X] [TopologicalSpace Y]
+
+example : Continuous (id : X → X) := by continuity
+
+example {f : X → Y} {g : Y → X} (hf : Continuous f) (hg : Continuous g) :
+  Continuous (fun x => f (g x)) := by continuity
+
+example {f : X → Y} {g : Y → X} (hf : Continuous f) (hg : Continuous g) :
+  Continuous (f ∘ g ∘ f) := by continuity
+
+example {f : X → Y} {g : Y → X} (hf : Continuous f) (hg : Continuous g) :
+  Continuous (f ∘ g) := by continuity
+
+-- Todo: more interesting examples when more algebra is ported
```

Comment:
I suggest adding tests for all the things that the mathlib3 version of `continuity` was designed to explicitly work around (e.g., issues with `Continuous.comp` and `continuous_const`).